qp_target_gene/pick_otus.py

```diff
@@ -55,6 +58,15 @@ def generate_pick_closed_reference_otus_cmd(filepaths, out_dir, parameters):
     # It should be only a single preprocessed fasta file
     seqs_fp = filepaths['preprocessed_fasta'][0]
 
+    if not test and is_zipfile(seqs_fp):
+        seqs_fp_fna = join(out_dir, 'seqs.fna')
+        cmd_ungz = 'gunzip -c %s > %s' % (seqs_fp, seqs_fp_fna)
+        std_out, std_err, return_value = system_call(cmd_ungz)
+        if return_value != 0:
+            error = ("Std out: %s\nStd err: %s\n\nCommand run was:\n%s"
+                     % (std_out, std_err, cmd_ungz))
+            raise RuntimeError(error)
```

Comment:
I think we could remove this block if instead we prepend the gunzip command to the pick_closed__reference_otus.py command. What do you think? That way error handling, etc is handled all in the same place.